src/main/resources/db/migration/V0003__SCRUM-2024.sql

```diff
@@ -0,0 +1,11 @@
+INSERT INTO subdomain (code, name, description)
+VALUES ('101', 'reference', 'Literature Reference');
+CREATE SEQUENCE subdomain_literature_reference_seq;
+
+INSERT INTO subdomain (code, name, description)
+VALUES ('102', 'resource', 'Literature Resource');
+CREATE SEQUENCE subdomain_literature_resource_seq;
```

Comment:
Actually you need to change the SEQ also.

Reply:
Changed and fixed a NULL pointer bug when searching for unexisting subdomains